hedera-node/hedera-app/src/test/java/com/hedera/node/app/state/merkle/TestLongCodec.java

```diff
@@ -23,11 +23,19 @@
 import edu.umd.cs.findbugs.annotations.Nullable;
 import java.util.Objects;
 
+/**
+ * This is a test-only class used exclusively by MerkleTestBase.
+ * <strong>This class should not be used elsewhere.</strong>
+ * The use of Codec implementations is not ideal, and we have the protobuf ProtoLong as
+ * a replacement.  When time permits this class needs to be removed and all usages replaced
+ * with ProtoLong and the ProtoLong.PROTOBUF codec.
+ * @deprecated Use ProtoLong and ProtoLong.PROTOBUF instead of Long and this codec.
+ */
```

Comment:
thank you for clarifying why this class exists!

Reply:
Thank @kimbor who suggested it 😁